src/components/_inputs/InputCategory.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <vue-bootstrap-typeahead
+    type="text"
+    autocomplete="off"
+    class="form-control"
+    :data="categories"
+    :serializer="({ name, parent }) => `${name} ${parent && parent.name ? `(${parent.name})` : ''}`"
```

Comment:
```suggestion
    :serializer="({ name, parent }) => `${name} ${(parent && parent.name ? '(' + parent.name + ')' : '')}`"
```

Você testou isso aí ? Parece que tem um erro de sintaxe aí 🙃

Reply:
rodei aqui, apareceu certinho em parenteses e tal